"report" command not working / is it still valid?

This looks like it initially was intended to be used perhaps towards some "verify" command; however, it is not used anywhere within the code and invoking it via the command line results in an exception.

```
$ ./wskdeploy report
----==== OpenWhisk Deployment Status ====----
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x13b928e]

goroutine 1 [running]:
github.com/apache/incubator-openwhisk-wskdeploy/cmd.printDeploymentInfo(0x0, 0x0, 0x1671240)
	/Users/Matt/go/src/github.com/apache/incubator-openwhisk-wskdeploy/cmd/report.go:86 +0x8e
github.com/apache/incubator-openwhisk-wskdeploy/cmd.glob..func6(0x16a2fa0, 0x16c2c58, 0x0, 0x0, 0x0, 0x0)
	/Users/Matt/go/src/github.com/apache/incubator-openwhisk-wskdeploy/cmd/report.go:49 +0x9e
github.com/spf13/cobra.(*Command).execute(0x16a2fa0, 0x16c2c58, 0x0, 0x0, 0x16a2fa0, 0x16c2c58)
	/Users/Matt/go/src/github.com/spf13/cobra/command.go:650 +0x457
github.com/spf13/cobra.(*Command).ExecuteC(0x16a31e0, 0xc4200aff78, 0x100670c, 0xc420072058)
	/Users/Matt/go/src/github.com/spf13/cobra/command.go:729 +0x339
github.com/spf13/cobra.(*Command).Execute(0x16a31e0, 0x0, 0x0)
	/Users/Matt/go/src/github.com/spf13/cobra/command.go:688 +0x2b
github.com/apache/incubator-openwhisk-wskdeploy/cmd.Execute()
	/Users/Matt/go/src/github.com/apache/incubator-openwhisk-wskdeploy/cmd/root.go:66 +0x40
main.main()
	/Users/Matt/go/src/github.com/apache/incubator-openwhisk-wskdeploy/main.go:26 +0x20

```

We should discuss if this is still valid/valuable... if it is we need to make it work OR remove it.  At minimum we should disable it in the near term.

--- a/cmd/report.go
+++ b/cmd/report.go
@@ -27,6 +27,7 @@ import (
 	"github.com/spf13/cobra"
 	"path"
 	"sync"
+    "os"
 )
 
 var wskpropsPath string
@@ -60,7 +61,7 @@ located under current user home.`,
 
 func init() {
 	RootCmd.AddCommand(reportCmd)
-	reportCmd.Flags().StringVarP(&wskpropsPath, "wskproppath", "w", ".", "path to wsk property file, default is to ~/.wskprops")
+	reportCmd.Flags().StringVarP(&wskpropsPath, "wskproppath", "w", path.Join(os.Getenv("HOME"), ".wskprops"), "path to wsk property file, default is to ~/.wskprops")
 
 	// Here you will define your flags and configuration settings.
 
@@ -76,7 +77,7 @@ func init() {
 
 var boldString = color.New(color.Bold).SprintFunc()
 
-func printDeploymentInfo(*whisk.Client) error {
+func printDeploymentInfo(client *whisk.Client) error {
 	//We currently list packages, actions, triggers, rules.
 	wg.Add(4)
 

--- a/deployers/whiskclient.go
+++ b/deployers/whiskclient.go
@@ -68,7 +68,7 @@ var CreateNewClient = func (config_input *whisk.Config) (*whisk.Client, error) {
 	var netClient = &http.Client{
 		Timeout: time.Second * utils.DEFAULT_HTTP_TIMEOUT,
 	}
-	return whisk.NewClient(netClient, clientConfig)
+	return whisk.NewClient(netClient, config_input)
 }
 
 // we are reading openwhisk credentials (apihost, namespace, and auth) in the following precedence order:

